Terminal.Gui/Drivers/WindowsDriver/WindowsOutput.cs

```diff
@@ -146,11 +146,16 @@ public WindowsOutput ()
                 throw new ApplicationException ($"Failed to set screenBuffer console mode, error code: {Marshal.GetLastWin32Error ()}.");
             }
 
-            // Force 16 colors if not in virtual terminal mode.
-            // BUGBUG: This is bad. It does not work if the app was crated without
-            // BUGBUG: Apis.
-            //ApplicationImpl.Instance.Force16Colors = true;
-
+            try
+            {
+                // Force 16 colors if not in virtual terminal mode.
+                (ApplicationImpl.Instance as ApplicationImpl)!.IsVirtualTerminal = false;
```

Comment:
Please re do this such that it does not depend on the static Application Instance.